MC/run/ANCHOR/anchorMC.sh

```diff
@@ -0,0 +1,181 @@
+#!/bin/bash
+
+# add distortion maps
+# https://alice.its.cern.ch/jira/browse/O2-3346?focusedCommentId=300982&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-300982
+#
+# export O2DPG_ENABLE_TPC_DISTORTIONS=ON
+# SCFile=$PWD/distortions_5kG_lowIR.root # file needs to be downloaded
+# export O2DPG_TPC_DIGIT_EXTRA=" --distortionType 2 --readSpaceCharge ${SCFile} "
+
+#
+# procedure setting up and executing an anchored MC 
+#
+
+# make sure O2DPG + O2 is loaded
+[ ! "${O2DPG_ROOT}" ] && echo "Error: This needs O2DPG loaded" && exit 1
+[ ! "${O2_ROOT}" ] && echo "Error: This needs O2 loaded" && exit 1
+
+# ------ CREATE AN MC CONFIG STARTING FROM RECO SCRIPT --------
+# - this part should not be done on the GRID, where we should rather
+#   point to an existing config (O2DPG repo or local disc or whatever)
+export ALIEN_JDL_LPMANCHORYEAR=${ALIEN_JDL_LPMANCHORYEAR:-2022}
+RUNNUMBER=${ALIEN_JDL_LPMRUNNUMBER:-517616}
+#INTERACTIONRATE=${INTERACTIONRATE:-2000}
+
+# get the async script (we need to modify it)
+# the script location can be configured with a JDL option
+
+ALIEN_JDL_LPMPRODUCTIONTAG_KEEP=$ALIEN_JDL_LPMPRODUCTIONTAG
+echo "Substituting ALIEN_JDL_LPMPRODUCTIONTAG=$ALIEN_JDL_LPMPRODUCTIONTAG with ALIEN_JDL_LPMANCHORPRODUCTION=$ALIEN_JDL_LPMANCHORPRODUCTION for simulating reco pass..."
+ALIEN_JDL_LPMPRODUCTIONTAG=$ALIEN_JDL_LPMANCHORPRODUCTION
+
+# ZDC causes issues for sim
+#export ALIEN_JDL_WORKFLOWDETECTORS=ITS,TPC,TOF,FV0,FT0,FDD,MID,MFT,MCH,TRD,EMC,PHS,CPV,HMP,ZDC,CTP
+export ALIEN_JDL_WORKFLOWDETECTORS=ITS,TPC,TOF,FV0,FT0,FDD,MID,MFT,MCH,TRD,EMC,PHS,CPV,HMP,CTP
+
+### async_pass.sh
+DPGRECO=$O2DPG_ROOT/DATA/production/configurations/asyncReco/async_pass.sh
+
+if [[ -f async_pass.sh ]]; then
+    chmod +x async_pass.sh
+    DPGRECO=./async_pass.sh
+else
+    cp -v $DPGRECO .
+fi
+
+if [[ ! -f setenv_extra.sh ]]; then
+    # cp -v ${DPGRECO%/*}/setenv_extra.sh .
+    cp -v $O2DPG_ROOT/DATA/production/configurations/asyncReco/setenv_extra.sh .
+fi
+
+echo "[INFO alien_async_pass.sh] Setting up DPGRECO to ${DPGRECO}"
+
+#settings that are MC-specific
+sed -i 's/GPU_global.dEdxUseFullGainMap=1;GPU_global.dEdxDisableResidualGainMap=1/GPU_global.dEdxSplineTopologyCorrFile=splines_for_dedx_V1_MC_iter0_PP.root;GPU_global.dEdxDisableTopologyPol=1;GPU_global.dEdxDisableGainMap=1;GPU_global.dEdxDisableResidualGainMap=1;GPU_global.dEdxDisableResidualGain=1/' setenv_extra.sh
+### ???
```

Comment:
@chiarazampolli : is this still needed?

Reply:
I don't know, maybe @wiechula does? Otherwise I need to check.

Reply:
I think this is a way to replace data TPC dEdx parameterizations (splines) wtht different ones tuned on MC.
Energy loss in MC is not exactly the same as in data  and gain calibration are not yet ready for MC (if I remember correctly)

Reply:
Is the setenv_extra.sh the default one from asyncReco https://github.com/AliceO2Group/O2DPG/blob/master/DATA/production/configurations/asyncReco/setenv_extra.sh ?

Reply:
Yes, should be: https://github.com/AliceO2Group/O2DPG/pull/1340/files#diff-ec62e7d374a5f942616197c7eb143f6ad3faba56988ed36c8f0f0e5b49f936e1R48

Reply:
The original comment should still be finally clarified before merging.